tests/test_cephes.py

```diff
@@ -0,0 +1,156 @@
+from __future__ import division
+import random
+from unittest import TestCase
+
+import numpy as np
+import scipy.special as special
+
+from cepy import bdtr
+from cepy import betaln
+from cepy import btdtr
+from cepy import gamln
+from cepy import gammainc
+from cepy import ndtri
+from cepy import pdtr
+
+
+np.seterr(all="ignore")
+
+
+def allClose(a, b):
```

Comment:
Have you considered using the built-in numpy.allclose function?

https://docs.scipy.org/doc/numpy/reference/generated/numpy.allclose.html